### PR TITLE
Add an external collaborator

### PR DESCRIPTION
**Pull request by andyrogers1973**
Gregory Whiting will be given access to yjaf-terraform repo

[Issue #2083](https://github.com/ministryofjustice/github-collaborators/issues/2083)

### CODE REVIEW

File: terraform/yjaf-terraform.tf [hunk 1]
```diff
@@ -32,5 +32,15 @@ module "yjaf-terraform" {
...
       review_after = "2025-03-14"
     },
+    {
+      github_user  = "gregi2n"
+      permission   = "admin"
+      name         = "Gregory Whiting"
+      email        = "greg.whiting@necsws.com"
+      org          = "NEC Software Solutions"
+      reason       = "Devops guys need access to make app/infra changes"
+      added_by     = "andy.rogers@digital,justice,gov.uk"
```
Comment (tamsinforbes) [inline terraform/yjaf-terraform.tf]:
```suggestion
      added_by     = "andy.rogers@digital.justice.gov.uk"
```

**Review by tamsinforbes — CHANGES_REQUESTED** [review]
Waiting clarification on access level required

**Review by AntonyBishop — DISMISSED** [review]
We can't be the `added_by` user. Access need to be authorised by David.Hall@yjb.gov.uk

**Review by tamsinforbes — APPROVED** [review]
Received approval via David Halls deputies.

**Pull request merged**